auth-web/src/components/auth/manage-business/CreateNewBusinessHelp.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div id="help-resolution" class="mb-12">
```

Comment:
Don't set whitespace here. The bounding box of components should be "tight" around it.

The parent should set the whitespace around a component. That way, this component can be used with different amounts of whitespace around it.

Reply:
Code has changed. Resolving.